src/Kris/LaravelFormBuilder/Form.php

```diff
@@ -618,6 +630,19 @@ public function __get($name)
     }
 
     /**
+     * Set the Event Dispatcher to fire Laravel events
+     *
+     * @param EventDispatcher $eventDispatcher
+     * @return $this
+     */
+    public function setEventDispatcher(EventDispatcher $eventDispatcher)
```

Comment:
`FormHelper` doesn't have the dispatcher. `FormBuilder` does. Move to Helper and always ask Helper to fire events?, or take the dispatcher from Helper and ask it directly?